### PR TITLE
 chore: add typescript issue template

### PR DESCRIPTION
**Pull request by shisama**
## Why

This shows TypeScript version explicitly.

## What

The version is written in typescript.md

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/js-sdk/blob/master/CONTRIBUTING.md)
- [ ] Updated documentation if it is required.
- [ ] Added tests if it is required.
- [ ] Passed `yarn lint` and `yarn test` on the root directory.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/typescript.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: "TypeScript Issue"
```
Comment (shisama) [inline .github/ISSUE_TEMPLATE/typescript.md:2]: What is an emoji for TypeScript issue?

File: .github/ISSUE_TEMPLATE/typescript.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- Babel: 0.0.0 <!-- if you use Babel. If not, delete this line. -->
+
+## Current Behavior
+
+<!-- What is the current behavior you saw?  -->
+
+## Expected Behavior
+
+<!-- What is the behavior you expected?  -->
+
```
Comment (shisama) [inline .github/ISSUE_TEMPLATE/typescript.md:26]: Are these necessary?

File: .github/ISSUE_TEMPLATE/typescript.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+<!-- Thank you for reporting a TypeScript issue! -->
+
+## Versions
+
+<!-- Which package do you want to report an issue for? -->
+<!-- What version did the error occur? -->
+- @kintone/xxx 0.0.0
+- TypeScript: 0.0.0
+- Babel: 0.0.0 <!-- if you use Babel. If not, delete this line. -->
```
Comment (shisama) [inline .github/ISSUE_TEMPLATE/typescript.md:17]: Is the Babel version necessary?

File: .github/ISSUE_TEMPLATE/typescript.md (hunk 1 above)
Comment (koba04) [reply .github/ISSUE_TEMPLATE/typescript.md:2]: What about 🙈 ? (That has no meaning, but cute)

File: .github/ISSUE_TEMPLATE/typescript.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+name: "TypeScript Issue"
```
Comment (koba04) [inline .github/ISSUE_TEMPLATE/typescript.md:2]:
```suggestion
name: "🙈 TypeScript Issue"
```

**Review by koba04 — CHANGES_REQUESTED** [review]
@shisama Thank you! I've put a suggestion for the emoji. Could you fix the error in the lint task in CI?

**Comment by shisama** [comment]
@koba04 Thank you for your suggestion. I committed it and fixed the lint error.

**Review by koba04 — APPROVED** [review]
👍👍👍

**Pull request merged**